vyxal/elements.py

```diff
@@ -3178,7 +3178,11 @@ def replace_until_no_change(lhs, rhs, other, ctx):
 def request(lhs, ctx):
     """Element ¨U
     (str) -> Send a GET request to a URL if online"""
-    x = urllib.request.urlopen(urlify(lhs)).read()
+
+    req = urllib.request.Request(
+        urlify(lhs), headers={"User-Agent": "Mozilla/5.0 Vyxal"}
```

Comment:
Might want to use the same user agent as Chrome

Reply:
All user agents [start with Mozilla](https://stackoverflow.com/questions/1114254/why-do-all-browsers-user-agents-start-with-mozilla)